Cargo.toml

```diff
@@ -5,34 +5,23 @@ authors = ["Zachary Golba <zachary.golba@postlight.com>"]
 edition = "2021"
 license = "MIT"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
 default = []
-serde = ["dep:serde", "dep:serde_json"]
+json = ["dep:serde", "dep:serde_json"]
 
 [dependencies]
 bytes = "1.7"
 futures-core = { version = "0.3", default-features = false }
 http = "1"
+hyper = { features = ["http1", "server"], version = "1.4" }
 hyper-util = { version = "0.1", features = ["tokio"] }
 percent-encoding = "2.3"
```

Comment:
I'm thinking that once we implement cookies, we'll probably want to feature flag `percent-encoding` and conditionally decode query params based on whether or not that feature flag is set.

Reply:
FWIW, I'm thinking `percent-encoding` will be a default feature since It's a pretty common use-case.